.github/workflows/upgrade-ui.yml

```
@@ -53,6 +53,27 @@ jobs:
         run: |
           docker compose -f "docker-compose.yml" up -d --build
           bash -c 'while [[ "$(curl -L -s -o /dev/null -w %{http_code} http://localhost/)" != "200" ]]; do sleep 5; done'
+
+      - name: Install Update assistant module
+        if: startsWith(matrix.PS_VERSION_START, '1.7')
+        env:
+          PS_VERSION: ${{ matrix.PS_VERSION_START }}
+          VERSION: ${{ matrix.PS_VERSION_START }}
+          BASE_VERSION: 1.7
+        run: |
+          docker exec -t prestashop chmod +x /usr/local/bin/install-update.sh
+          docker exec -t prestashop /usr/local/bin/install-update.sh
```

* You should add the execution flag on the script and commit the change
* The command should be run as `www-data` to avoid issues with permissions

```suggestion
          docker exec --user=www-data -t prestashop /usr/local/bin/install-update.sh
```